Configuration/DataProcessing/test/RunPromptReco.py

```diff
@@ -29,6 +29,8 @@ def __init__(self):
         self.alcaRecos = None
         self.PhysicsSkims = None
         self.dqmSeq = None
+        self.setRepacked = False
+        self.isRepacked = False
```

Comment:
@slava77 - I think this would be more consistent if the default of isRepacked here is True (as that is default elsewhere in the PR)